com.xargon.retrodeck.yml

```
@@ -0,0 +1,643 @@
+app-id: com.xargon.retrodeck
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+# Needed for Yuzu
+base: io.qt.qtwebengine.BaseApp
+base-version: 5.15-21.08
+# /Needed for Yuzu
+command: retrodeck.sh
+
+finish-args:
+   - --socket=x11
+   - --socket=wayland
+   - --socket=pulseaudio
+   - --share=ipc
+   - --share=network
+   - --device=all
+   - --filesystem=~/retrodeck:create
+   - --filesystem=/run/media
```

```suggestion
   - --filesystem=/run/media
   - --filesystem=/media
```

Sadly this is also need for removable media.